carla_vehicle_spawner/carla_vehicle_spawner/create_vehicles.py

```diff
@@ -0,0 +1,54 @@
+import carla
+import random
+import time
+import rclpy
+from rclpy.node import Node
+
+class VehicleSpawner(Node):
+    def __init__(self):
+        super().__init__('vehicle_spawner')
+        self.client = carla.Client('localhost', 2000)
```

Comment:
Is this the port for carla?  If so could we access the ROS_DOMAIN_ID in the same way we use it to spawn carla with a unique port?  See Line 22 of launch.carla_interface.py in the root folder.